lib/screens/images_to_pdf.dart

```diff
@@ -17,7 +18,10 @@ class ImageToPdf extends StatelessWidget {
                     padding: const EdgeInsets.symmetric(horizontal: 4),
                     child: FloatingActionButton.extended(
                       heroTag: 'export',
-                      onPressed: _imageToPdfController.exportToPdf,
+                      onPressed: () {
+                        _imageToPdfController.exportToPdf();
+                        Get.to(HomeScreen());
```

Comment:
why navigate to home is that good UX?

Reply:
> why navigate to home is that good UX?

its beacuase user is  unknown about his / her pdgf is creatded or not

previously the page remain static but 

this may atleast changes the page

Reply:
show toast instead

Reply:
ok sure